<launch>
    <arg name="map_file" default="$(find stero_mobile_init)/maps/labirynt/map.yaml"/>

    <include file="$(find stero_mobile_init)/launch/tiago_simple.launch">
        <arg name="world" value="labirynt"/>

    </include>

    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)"/>

    <include file="$(find amcl)/examples/amcl_diff.launch" />

    <node pkg="stero_mobile_init" type="navigation_node" respawn="false" name="navigation_node" output="screen">
        <rosparam file="$(find stero_mobile_init)/navigation/map_params/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
        <rosparam file="$(find stero_mobile_init)/navigation/map_params/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find stero_mobile_init)/navigation/map_params/local_costmap_params.yaml" command="load"/>
        <rosparam file="$(find stero_mobile_init)/navigation/map_params/global_costmap_params.yaml" command="load" /> 
        <rosparam file="$(find stero_mobile_init)/navigation/map_params/base_local_planner_params.yaml" command="load" ns="my_dwa_planner"/>
        </node>

    <node name="labirynttrans" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 map odom 100"/>
    <!-- <param name="sensor_param" value="param_value" /> -->

</launch> 